reachy_sdk_server/reachy_sdk_server/grpc_server/hand.py

```diff
@@ -0,0 +1,201 @@
+import grpc
+import numpy as np
+import rclpy
+
+from control_msgs.msg import DynamicJointState, InterfaceValue
+from google.protobuf.empty_pb2 import Empty
+from google.protobuf.wrappers_pb2 import FloatValue
+
+from ..abstract_bridge_node import AbstractBridgeNode
+from ..parts import Part
+
+from reachy2_sdk_api.hand_pb2_grpc import (
+    add_HandServiceServicer_to_server,
+)
+from reachy2_sdk_api.hand_pb2 import (
+    Force,
+    Hand,
+    HandPosition,
+    HandPositionRequest,
+    HandState,
+    HandStatus,
+    HandTemperatures,
+    JointsLimits,
+    ListOfHand,
+    ParallelGripperPosition,
+    SpeedLimitRequest,
+)
+from reachy2_sdk_api.part_pb2 import (
+    PartId,
+)
+
+
+class HandServicer:
+    def __init__(
+        self,
+        bridge_node: AbstractBridgeNode,
+        logger: rclpy.impl.rcutils_logger.RcutilsLogger,
+    ) -> None:
+        self.bridge_node = bridge_node
+        self.logger = logger
+
+        self.hands = self.bridge_node.parts.get_by_type("hand")
+
+    def register_to_server(self, server: grpc.Server):
+        self.logger.info("Registering 'HandServiceServicer' to server.")
+        add_HandServiceServicer_to_server(self, server)
+
+    def get_hand(self, hand: Part, context: grpc.ServicerContext) -> Hand:
+        return Hand(
+            part_id=PartId(name=hand.name, id=hand.id),
+        )
+
+    def get_hand_part_from_part_id(
+        self, part_id: PartId, context: grpc.ServicerContext
+    ) -> Part:
+        part = self.bridge_node.parts.get_by_part_id(part_id)
+
+        if part is None:
+            context.abort(grpc.StatusCode.NOT_FOUND, f"Part not found (id={part_id}).")
+
+        if part.type != "hand":
+            context.abort(
+                grpc.StatusCode.INVALID_ARGUMENT,
+                f"Part '{part_id}' is not an hand.",
+            )
+
+        return part
+
+    def GetAllHands(self, request: Empty, context: grpc.ServicerContext) -> ListOfHand:
+        return ListOfHand(hand=[self.get_hand(hand, context) for hand in self.hands])
+
+    def GetHandState(self, request: PartId, context: grpc.ServicerContext) -> HandState:
```

Comment:
To be consistent with the API changes
```suggestion
    def GetState(self, request: PartId, context: grpc.ServicerContext) -> HandState:
```